package.json

```diff
@@ -3,11 +3,14 @@
   "version": "0.0.3",
   "description": "The only GroupMe API library that isn't a million years old.",
   "main": "dist/index.js",
-  "scripts": {
-    "test": "mocha \"tests/**/*.js\" --forbid-only --bail --recursive --require tests/hooks.js",
-    "build": "npx tsc",
+  "scripts": {    
+    "clean": "rm -rf dist",
+    "compile": "npm run clean && tsc -p .",
```

Comment:
what is the `-p .` option for?

Reply:
It just specifies the directory, in case tsc changes the default behavior with no args for some weird reason

Reply:
interesting, ok ty